src/Layout/Presentation/Presentation.tsx

```diff
@@ -18,6 +18,7 @@ import { LagueDhontResult, PartyResult, DistrictResult, AlgorithmType } from "..
 import { PresentationType, DisproportionalityIndex } from "./presentation-models";
 import { ElectionComparison } from "./ElectionOverview/ElectionComparison";
 import { checkExhaustively } from "../../utilities";
+import { Dictionary } from "utilities/dictionary";
```

Comment:
Codacy found an issue: [Submodule import paths from this package are disallowed; import from the root instead](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)